### PR TITLE
Update to PHP CS Fixer 2

### PR DESCRIPTION
**Pull request by dunglas**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

### CODE REVIEW

File: tests/Bridge/Symfony/Bundle/DependencyInjection/Compiler/FilterPassTest.php [hunk 1]
```diff
@@ -12,7 +12,6 @@
 namespace ApiPlatform\Core\Tests\Bridge\Symfony\Bundle\DependencyInjection\Compiler;
 
 use ApiPlatform\Core\Bridge\Symfony\Bundle\DependencyInjection\Compiler\FilterPass;
-use ApiPlatform\Core\Exception\RuntimeException;
```
Comment (soyuka) [inline tests/Bridge/Symfony/Bundle/DependencyInjection/Compiler/FilterPassTest.php]: Why this change?

File: tests/Api/ResourceClassResolverTest.php [hunk 2]
```diff
@@ -49,7 +48,7 @@ public function testGetResourceClassWithNoClassName()
     }
 
     /**
-     * @expectedException InvalidArgumentException
```
Comment (soyuka) [inline tests/Api/ResourceClassResolverTest.php]: Here too, is this wanted?

File: tests/Bridge/Symfony/Bundle/DependencyInjection/Compiler/FilterPassTest.php (hunk 1 above)
Comment (dunglas) [inline tests/Bridge/Symfony/Bundle/DependencyInjection/Compiler/FilterPassTest.php]: It's unused but it's a bug in the test, I'll open a new PR to fix the test.

File: tests/Api/ResourceClassResolverTest.php (hunk 2 above)
Comment (dunglas) [inline tests/Api/ResourceClassResolverTest.php]: Same here, it's invalid, I'll fix this in another PR.
